db/primary_migrate/20221011151135_add_issuer_to_profile.rb

```diff
@@ -0,0 +1,5 @@
+class AddIssuerToProfile < ActiveRecord::Migration[7.0]
+  def change
+    add_column :profiles, :initiating_issuer, :string, null: true
```

Comment:
I don't have a strong feeling whether we should, but would we care to establish a relation between this field and `ServiceProvider` so that we could reference the object via e.g. `profile.initiating_service_provider.friendly_name` ?

Reply:
Yup - done in fb5ad96a8